cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1300,20 +1300,31 @@ declare namespace Deno {
    */
   export namespace telemetry {
     /**
-     * A SpanExporter compatible with OpenTelemetry.js
-     * https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_sdk_trace_base.SpanExporter.html
+     * A TracerProvider compatible with OpenTelemetry.js
+     * https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_api.TracerProvider.html
      * @category Telemetry
      * @experimental
      */
-    export class SpanExporter {}
+    // deno-lint-ignore no-explicit-any
+    export const TracerProvider: any;
 
     /**
      * A ContextManager compatible with OpenTelemetry.js
      * https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_api.ContextManager.html
      * @category Telemetry
      * @experimental
      */
-    export class ContextManager {}
+    // deno-lint-ignore no-explicit-any
+    export const ContextManager: any;
```

Comment:
if these are supposedly singletons, should they still be PascalCase?

Reply:
actually, i'll change it to regular camelCase.